### PR TITLE
Updated FontAwesome icons from 4.4.0 to 4.5.0

### PR DESCRIPTION
**Pull request by KhairulA**

### CODE REVIEW

**Comment by mjolnic** [comment]
thanks @KhairulA  but I cannot accept this PR. You changed more than just FontAwesome icons: reformatted the whole js code.

**Comment by KhairulA** [comment]
Hi @mjolnic, most of the js changes are by Grunt's uglify. My grunt-cli is v1.1.0, npm is 2.14.20.

Modified files:
- Gruntfile.js
- src/js/iconpicker.js

Font Awesome changes:
- extension/css/font-awesome.css
- extension/fonts/FontAwesome.otf
- extension/fonts/fontawesome-webfont.svg
- extension/fonts/fontawesome-webfont.ttf
- extension/fonts/fontawesome-webfont.woff
- extension/fonts/fontawesome-webfont.woff2
- extension/js/fontawesome-iconpicker.js

**Comment by mjolnic** [comment]
You changed the Gruntfile configuration, why?

File: Gruntfile.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ module.exports = function(grunt) {
         uglify: {
             distMin: {
                 options: {
-                    compress: true,
+                    compress: {},
```
Comment (mjolnic) [inline Gruntfile.js]: why is this changed?

File: Gruntfile.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ module.exports = function(grunt) {
             },
             dist: {
                 options: {
-                    compress: false,
+                    compress: {},
```
Comment (mjolnic) [inline Gruntfile.js]: also this
